Docs/source/pages/commands.md

```diff
@@ -3161,8 +3161,8 @@ Wait until a property has a specific value and returns the value of the given co
 | propertyName  | string | Yes      | Name of the property of which value you want. If the property is an array you can specify which element of the array to return by doing property[index], or if you want a property inside of another property you can get by doing property.property2 for example position.x.                                                           |                                                                                                                                   
 | propertyValue  | T | Yes       | The value that property shoud have.                             
 | assemblyName  | string | Yes       | The name of the assembly containing the component.                                                                                                                           
-| timeout     | double             | No       | The number of seconds that it will wait for property.                                                                                                                            
-| interval    | double             | No       | The number of seconds after which it will try to find the object again. The interval should be smaller than timeout.                                                                                                                                                                                                                                                                                       |
+| timeout     | double             | No       | The number of seconds that it will wait for  property. The default value is 20 seconds.                                                                                                                            
```

Comment:
```suggestion
| timeout     | double             | No       | The number of seconds that it will wait for the property. The default value is 20 seconds.                                                                                                                            
```